config/nrfconnect/chip-module/Kconfig

```diff
@@ -147,9 +147,9 @@ config CHIP_FACTORY_DATA_ROTATING_DEVICE_UID_MAX_LEN
 
 config CHIP_FACTORY_DATA_WRITE_PROTECT
 	bool "Enable Factory Data write protection"
-	select FPROTECT
+	imply FPROTECT
```

Comment:
Is it necessary? I mean if `fprotect` is the only protection mechanism we have but it's unavailable on nRF54, we should rather not enable `CHIP_FACTORY_DATA_WRITE_PROTECT` instead of pretending it's supported.